P1 - Training.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    LOSS_FN = losses.normalized_L1_loss
</pre><p>Your capitalization is a bit mixed. It is fine to make global and constants capital, however below and above there are other variables that aren't. Either is fine for me as long as consistent :) </p><p> </p><p>(see also btw to know about differences to C++ https://peps.python.org/pep-0008/#descriptive-naming-styles )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gomezzz/geodesyNets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>